service/README.md

```diff
@@ -55,6 +55,13 @@ Run integration tests (needs to have docker daemon running):
 ./gradlew integrationTest
 ```
 
+To start dev server connected to integration test database (suitable for e.g. running E2E tests without cluttering
+ development data):
+```sh
+./gradlew bootRunTest
+```
+Edit `gradle.properties` to run flyway commands against integration test database.
```

Comment:
Enable running flyway commands against integration test database in `gradle.properties´" (I think this is better because it gives the proper impression that dev does not need to know how to do the db config change, just needs to use existing configuration)